src/bin/vip-logs.js

```diff
@@ -201,30 +201,36 @@ command( {
 	envContext: true,
 	module: 'logs',
 } )
-	.option( 'type', 'The type of logs to be returned: "app" or "batch"', 'app' )
-	.option( 'limit', 'The maximum number of log lines', 500 )
-	.option( 'follow', 'Keep fetching new logs as they are generated' )
-	.option( 'format', 'Output the log lines in CSV or JSON format', 'table' )
+	.option(
+		'type',
+		'Specify the type of Runtime Logs to retrieve. Accepts "batch" (only valid for WordPress environments) or "app" (default).'
+	)
+	.option(
+		'limit',
+		'Set a maximum number of entries to retrieve. Accepts an integer value between 1 and 5000.',
+		500
+	)
+	.option( 'follow', 'Output new entries as they are generated.' )
+	.option( 'format', 'Output entries in CSV or JSON format.', 'table' )
 	.examples( [
 		{
-			usage: 'vip @mysite.production logs',
-			description: 'Get the most recent app logs',
+			usage: 'vip @example-app.production logs',
+			description:
+				'Retrieve up to 500 of the most recent entries of application Runtime Logs from web containers.',
```

Comment:
Do you think we should mention that 500 is the default limit?
https://github.com/Automattic/vip-cli/pull/2089/files#diff-2842aeea5897751928ed910d842cd542768c354d43d746f1877afab1869eac9fR210

Reply:
The default setting for the `--limit` option can be found in the option's description when you query the help menu in the terminal:
`-l, --limit <n>       Set a maximum number of entries to retrieve. Accepts an integer value between 1 and 5000. (defaults to 500)`

Any limitations or instructions for how to use a subcommand or an option belong in the description of the subcommand or the option. The usage examples are intended to illustrate possible ways to use various combinations of the subcommands and their supported options.

Reply:
Oh, I didn't realize that the system appends `(defaults to XXX)`.